contracts/staking/ValidatorMessages.sol

```diff
@@ -19,28 +19,81 @@ library ValidatorMessages {
     // The P-Chain uses a hardcoded codecID of 0 for all messages.
     uint16 internal constant CODEC_ID = 0;
 
+    // The P-Chain signs a SubnetConversion message that is used to verify for the Subnet's initial validators.
+    uint32 internal constant SUBNET_CONVERSION_MESSAGE_TYPE_ID = 0;
+
     // Subnets send a RegisterSubnetValidator message to the P-Chain to register a validator.
-    uint32 internal constant REGISTER_SUBNET_VALIDATOR_MESSAGE_TYPE_ID = 0;
+    uint32 internal constant REGISTER_SUBNET_VALIDATOR_MESSAGE_TYPE_ID = 1;
 
     // Subnets can send a SetSubnetValidatorWeight message to the P-Chain to update a validator's weight.
     // The P-Chain responds with a SetSubnetValidatorWeight message acknowledging the weight update.
-    uint32 internal constant SET_SUBNET_VALIDATOR_WEIGHT_MESSAGE_TYPE_ID = 1;
+    uint32 internal constant SET_SUBNET_VALIDATOR_WEIGHT_MESSAGE_TYPE_ID = 2;
 
     // The P-Chain responds with a RegisterSubnetValidator message indicating whether the registration was successful
     // for the given validation ID.
-    uint32 internal constant SUBNET_VALIDATOR_REGISTRATION_MESSAGE_TYPE_ID = 2;
+    uint32 internal constant SUBNET_VALIDATOR_REGISTRATION_MESSAGE_TYPE_ID = 3;
 
     // The P-Chain responds with a SubnetValidatorWeightUpdateMessage message indicating whether the weight update was successful
     // for the given validation ID.
-    uint32 internal constant SUBNET_VALIDATOR_WEIGHT_UPDATE_MESSAGE_TYPE_ID = 3;
+    uint32 internal constant SUBNET_VALIDATOR_WEIGHT_UPDATE_MESSAGE_TYPE_ID = 4;
 
     // The Subnet will self-sign a ValidationUptimeMessage to be provided when a validator is initiating
     // the end of their validation period.
-    uint32 internal constant VALIDATION_UPTIME_MESSAGE_TYPE_ID = 4;
+    uint32 internal constant VALIDATION_UPTIME_MESSAGE_TYPE_ID = 5;
 
     // TODO: The implementation of these packing and unpacking functions is neither tested nor optimized at all.
     // Full test coverage should be provided, and the implementation should be optimized for gas efficiency.
 
+    function packSubnetConversionMessage(bytes32 subnetConversionID)
```

Comment:
Can you add natspec comments to these methods to match the others? The message specs are especially useful.